### PR TITLE
WEBEU2 - Preprocessing I - Sulaiman Abidemi 🇳🇬

### PR DESCRIPTION
**Pull request by abidex4yemi**
### What does this commit do?

* Add header markup
* Add css styles for header markup

Screenshot 2019-05-01 at 3 41 11 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,131 @@
...
-
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
+    <title>My Resume</title>
+    <link rel="stylesheet" href="css/index.css" />
+    <!--[if lt IE 9]>
```
Comment (maximesalomon) [inline index.html:7]: Don't forget the `<meta name="viewport" content="width=device-width, initial-scale=1.0">`

File: index.html [hunk 2]
```diff
@@ -1,19 +1,131 @@
...
+    <main class="main-section">
+      <div class="container">
+        <article class="showcase">
+          <section class="dev-intro">
+            <img
+              src="./img/jonny_img.png"
+              alt="johnny image"
+              class="showcase-img"
+            />
+            <h2>Jonny Appleseed-Web Dev</h2>
```
Comment (maximesalomon) [inline index.html:40]: Why not a `<h1>` here?

File: index.html [hunk 3]
```diff
@@ -1,19 +1,131 @@
...
+              Occasionally I have time to work on side projects, if you are
+              interested, let me know by sign in up on this website
+            </p>
+            <a href="#!">Lets Team Up</a>
+          </section>
+        </article>
+        <!--/.showcase-->
+
+        <section class="dev-details">
+          <article class="skills">
```
Comment (maximesalomon) [inline index.html:53]: Don't use too much the `<article>` tag - it had a defined use case. Read more on it https://www.w3schools.com/tags/tag_article.asp

File: less/index.less [hunk 4]
```diff
@@ -1,59 +1,212 @@
-// Is this working?
-/* http://meyerweb.com/eric/tools/css/reset/ 
-   v2.0 | 20110126
-   License: none (public domain)
-*/
+@import "reset.less";
```
Comment (maximesalomon) [inline less/index.less:1]: 👍

File: less/index.less [hunk 5]
```diff
@@ -1,59 +1,212 @@
...
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
-}
-/* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+/*===============================
```
Comment (maximesalomon) [inline less/index.less:3]: Great job with your comments - it's way easier to parse the file this way!

File: less/index.less [hunk 6]
```diff
@@ -1,59 +1,212 @@
...
+  display: flex;
+  align-items: baseline;
+  justify-content: @justifyContent;
 }
-body {
-	line-height: 1;
+
+// base font sizes
+@defaultFontSize: 100%;
+@mobileFontSize: 150%;
```
Comment (maximesalomon) [inline less/index.less:34]: Not sure how would you use it. What is your thinking behind this?

File: less/index.less [hunk 7]
```diff
@@ -1,59 +1,212 @@
...
+
+.header {
+  padding: @space-20 0;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+
+.nav {
```
Comment (maximesalomon) [inline less/index.less:65]: Great use of nesting!

File: less/reset.less [hunk 8]
```diff
@@ -0,0 +1,146 @@
+// Is this working?
```
Comment (maximesalomon) [inline less/reset.less:1]: Good job using an other file for your reset

File: less/index.less [hunk 9]
```diff
@@ -1,59 +1,212 @@
...
 
+.page-footer {
+  text-align: center;
+  margin-top: @space-5;
+
+  div {
+    background: @primary-bg-color;
+    padding: @space-5 0;
+  }
+}
```
Comment (maximesalomon) [inline less/index.less:212]: No media queries?

**Review by maximesalomon — COMMENTED** [review]
Good job on this Yemi 👍 

- You created a reset.less file and use import ✅
- You used variables to set fonts & colors ✅
- You found a good balance with your nesting ✅
- You did not use mixins ⚠️
 - You did not use media queries ⚠️

Make sure you read the README fully before starting your projects

You got this 💪

File: index.html (hunk 1 above)
Comment (abidex4yemi) [reply index.html:7]: Sure couldn' t, due to my bad network connectivity I couldn't make the site responsive

File: index.html (hunk 2 above)
Comment (abidex4yemi) [reply index.html:40]: Alright I will implement this

File: index.html (hunk 3 above)
Comment (abidex4yemi) [reply index.html:53]: Sure, I will read about it.

File: less/index.less (hunk 9 above)
Comment (abidex4yemi) [reply less/index.less:212]: Yes, I will implement this.

File: less/index.less (hunk 6 above)
Comment (abidex4yemi) [reply less/index.less:34]: it was copied from the class lecture repo, I forgot to remove it.

**Pull request merged**